### DISCUSSION TITLE
update the version

### DISCUSSION THREAD

**Discussion by XYW-TJU** [deepmodeling/reacnetgenerator#1766] [Q&A] [answered]
Hello! Mr. Zeng
I have already installed the old version on my linux system now, so how do I implement the version update.
On top of Anaconda3-2022.05-Linux-x86_64.sh installed, the installation command is as follows

conda create -n reax python=3.6
conda activate reax
pip install reacnetgenerator

Can you please tell me how to update the version?

**Comment by Eipgen** [comment, marked as answer]
Maybe you need to follow the instruction use python 3.7 later.
